bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -78,25 +79,38 @@ pub mod pallet {
 		/// To withdraw and deposit an asset.
 		type AssetTransactor: TransactAsset;
 
-		#[cfg(feature = "runtime-benchmarks")]
-		type Helper: BenchmarkHelper<Self::RuntimeOrigin>;
+		/// To charge XCM delivery fees
+		type XcmExecutor: ExecuteXcm<Self::RuntimeCall>;
 
+		/// Fee asset for the execution cost on ethereum
 		type FeeAsset: Get<Location>;
```

Comment:
Strictly speaking, this a location, not a fee.
```suggestion
		/// Location of the remote ethereum network
		type EthereumLocation: Get<Location>;
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/ac5090e505ad9a41f931e0b7004b362d602d5782